components/form-toggle/style.scss

```diff
@@ -52,11 +38,35 @@ $toggle-border-width: 2px;
 		border-radius: 50%;
 		background: $dark-gray-500;
 		transition: 0.1s transform ease;
+	}
 
-
-		.components-form-toggle.is-checked & {
+	&.is-checked {
+		&:after {
 			background-color: $white;
 			transform: translateX( $toggle-width - ( $toggle-border-width * 4 ) - ( $toggle-height - ( $toggle-border-width * 4 ) ) );
 		}
+
+		&:before {
+			background-color: $blue-medium;
+			border: $toggle-border-width solid $blue-medium;
+		}
 	}
 }
+
+.components-form-toggle__input[type=checkbox] {
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	opacity: 0;
+	margin: 0;
+	padding: 0;
+	z-index: z-index( '.components-form-toggle__input' );
```

Comment:
yes, this is a trick to allows the `onChange` handler to trigger when we click on the invisible input.